src/jobs/NotificationTrigger/notificationTime.js

```diff
@@ -1,12 +1,17 @@
 const moment = require('moment')
 const momentTz = require('moment-timezone')
-const { NOTIFICATION_END_TIME, NOTIFICATION_START_TIME } = require('./config')
-const verifyNoticiableTime = (currentTime, startTime = NOTIFICATION_START_TIME, endTime = NOTIFICATION_END_TIME) => {
-	const currentNumericTime = Number(currentTime.replace(':', ''))
-	if (currentNumericTime >= startTime && currentNumericTime <= endTime) {
-		return true
-	}
-	return false
+const NOTIFICATION_TIMES = ['6:00','9:00','12:00','15:00','18:00','21:00'];
```

Comment:
@ShivaAryal Can you fetch the times from environment variables?

Reply:
Yeah done